src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -249,6 +250,10 @@ async function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourc
   const existingPostRun = moduleConfig.postRun || [];
   (moduleConfig as any).preloadPlugins = [];
 
+  const totalResources = Object.keys(resources.assembly).length + Object.keys(resources.pdb || {}).length + /* dotnet.wasm */ 1;
+  Blazor.webAssemblyProgressService?.setTotalResources(totalResources);
+
+
```

Comment:
```suggestion
  const totalResources = Object.keys(resources.assembly).length + Object.keys(resources.pdb || {}).length + /* dotnet.wasm */ 1;
  Blazor.webAssemblyProgressService?.setTotalResources(totalResources);

```

Reply:
Do we also need to include `dotnetTimeZoneResourceName` and `icuDataResource` here?